filebeat/prospector/prospector.go

```diff
@@ -21,6 +21,7 @@ var (
 )
 
 type Prospector struct {
+	harvesterCounter uint64         // Must be 8-byte aligned. Ensured if first field in struct
```

Comment:
As it is quite likely that this is changed by accident in the future, I suggest to add a comment on top of the variable with the following content:

// harvesterCount MUST be first field in struct. See https://github.com/golang/go/issues/599